<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVeteransSetupDialog</class>
 <widget class="QDialog" name="ReportVeteransSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Параметры отчета</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата начала периода</string>
   </property>
  </widget>
  <widget class="CDateEdit" name="edtEndDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="CDateEdit" name="edtBegDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата окончания периода</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Подразделение</string>
   </property>
  </widget>
  <widget class="COrgStructureComboBox" name="cmbOrgStructure">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="CPersonComboBoxEx" name="cmbPerson">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>109</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPerson">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Врач</string>
   </property>
   <property name="buddy">
    <cstring>cmbPerson</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblAge">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>612</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Во&amp;зраст с</string>
   </property>
  </widget>
  <widget class="CRBComboBox" name="cmbEventType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblEventType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>199</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Тип обращения</string>
   </property>
   <property name="buddy">
    <cstring>cmbEventType</cstring>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVeteransSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVeteransSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
